data/zilliqa/assets/zil1c6akv8k6dqaac7ft8ezk5gr2jtxrewfw8hc27d/meta.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "DuckDuck",
   "symbol": "Duck",
-  "product": false,
+  "product": true,
```

Comment:
There's not much activity on chain + I tried the new app without success (cannot load any duck), so remove the product prop from the 3 tokens still and we'll revisit later

Reply:
@balthazar I can remove product that's fine.
Ducks should load, you need to sign into to the wallet? What about the NFTDeployer/Viewer? 
There's about to be 10 projects launching with viewing functionality probably expecting this to be set to true